doc/devel/documenting_mpl.rst

```diff
@@ -54,9 +54,36 @@ requirements that are needed to build the documentation. They are listed in
 
 .. note::
 
-  * You'll need a minimal working LaTeX distribution for many examples to run.
+  * You will need a minimal working LaTeX and Graphviz distribution for many
+    examples to run.
   * `Graphviz <http://www.graphviz.org/Download.php>`_ is not a Python package,
     and needs to be installed separately.
+  * Create a seprate environment using conda/venv.
+  
+  .. code-block:: sh
+
+   virtualenv venv
+   source /path/to/venv/bin/activate
+  
+  * You will need to install the addtional requirements.
+  
+  .. code-block:: sh
+
+   pip install -r requirements/doc/doc-requirements.txt
+   
+  * You will need to uninstall the conda/pip version of Matplotlib first, before
+    installing the dev version.
+    
+  .. code-block:: sh
+    
+    conda uninstall matplotlib
```

Comment:
if you are in a new env then matplotlib won't be installed in it.

Reply:
Should I remove this ?